cg/store/api/base.py

```diff
@@ -127,3 +131,77 @@ def _get_latest_analyses_for_cases_query(self) -> Query:
                 self.Analysis.started_at == case_and_date_subquery.c.started_at,
             ),
         )
+
+    def _get_filtered_case_query(
+        self,
+        case_action: Optional[str],
+        customer_id: str,
+        data_analysis: str,
+        days: int,
+        exclude_customer_id: bool,
+        internal_id: str,
+        name: str,
+        priority: str,
+        sample_id: str,
+    ) -> Query:
+        cases_query: Query = self._get_query(table=Family)
+        filter_functions: List[Callable] = []
+
+        filter_case_order_date = None
+        if days != 0:
+            filter_case_order_date = datetime.datetime.now() - datetime.timedelta(days=days)
```

Comment:
I think there is a utility function get_date_days_ago for this type of date handling